src/main.rs

```diff
@@ -135,9 +134,7 @@ pub async fn main() {
 		log::info!("IPFS backed application client: {}\t{:?}", peer_id, addrs);
 	}
 
-	let block_header = rpc::get_chain_header(&cfg.full_node_rpc)
-		.await
-		.expect("failed to get latest block header of chain");
+	let block_header = rpc::get_chain_header(&cfg.full_node_rpc).await?;
```

Comment:
Previously we used to show error: `failed to get latest block header of chain`. Is that still preserved?

Reply:
It will show something like `failed to build HTTP POST request object(get_chainHeader)`. Or we can change to old one